servers/graphql-kotlin-server/src/main/kotlin/com/expediagroup/graphql/server/execution/GraphQLContextFactory.kt

```diff
@@ -16,23 +16,16 @@
 
 package com.expediagroup.graphql.server.execution
 
-import com.expediagroup.graphql.generator.execution.GraphQLContext
+const val GRAPHQL_COROUTINE_CONTEXT_KEY = "GRAPHQL_COROUTINE_CONTEXT"
```

Comment:
Is this supposed to be `GRAPHQL_COROUTINE_SCOPE`? Maybe we should just use the value defined in the generator package so we don't have it defined in two locations

Reply:
Actually I'm thinking of dropping the String key in favor of just using classes as keys.